apps/site/scripts/orama-search/get-documents.mjs

```diff
@@ -63,11 +78,18 @@ export const siteContent = [...pageData, ...apiData]
 
     const siteSection = pathname.split('/').shift();
     const subSections = splitIntoSections(markdownContent);
-
     return subSections.map(section => {
+      const path = pathname + '#' + slug(section.pageSectionTitle);
+
       return {
-        path: pathname + '#' + slug(section.pageSectionTitle),
-        siteSection,
+        path,
+        formattedPath: getFormattedPath({
+          path,
+          title: section.pageSectionTitle,
+        }),
+        siteSection: uppercaseFirst(siteSection),
+        pageLink:
+          siteSection.toLowerCase() === 'docs' ? `/${path}` : `/en/${path}`,
```

Comment:
FYI instead of hardcoding `/en` retrieve the current language code :)

Reply:
Are you asking this to have multi-language search?

Reply:
All these pages will exist in all languages that we support, even if fallbacking to English translations.

Hence the link instead of being hardcoded to /en/ should point to the current language code.

Reply:
(Except API docs, that one will never have language prefixes)

Reply:
@ovflowd we verified and would very much appreciate if we could handle this in a separate PR

Reply:
I think this needs to be handled on this PR -- we can't hardcode `/en` here 😅

Reply:
Hi @ovflowd 
we were actually not using this property anymore as paths are not build dynamically. I'm removing it right now